client/app/hosting/ssl/order/hosting-order-ssl.controller.js

```diff
@@ -88,14 +88,22 @@ angular.module("App").controller(
 
                 this.Hosting.createSsl(this.$stateParams.productId, data.certificate, data.key, data.chain)
                     .then(() => this.Alerter.success(this.$scope.tr("hosting_dashboard_ssl_generate_success"), this.$scope.alerts.main))
-                    .catch((err) => this.Alerter.alertFromSWS(this.$scope.tr("hosting_dashboard_ssl_order_error"), err, this.$scope.alerts.main))
+                    .catch((err) => this._handleSslError(err))
                     .finally(() => {
                         this.$scope.loadSsl();
                         this.$scope.resetAction();
                     });
             }
         }
 
+        _handleSslError (err) {
+            let message = this.$scope.tr("hosting_dashboard_ssl_order_error");
+            if (err.message === "No attached domain with ssl enabled or no attached domain that redirect on hosting IPs, please use hosting IPs in your domain zone") {
```

Comment:
Forget :p

We should display a generic message in french and the SSL error in english in parenthesis.

To translate error, we are talking about error code etc ! We should find a generic solution ;)

Reply:
:+1: